docs/source/workflow.rst

```diff
@@ -84,7 +84,10 @@ configuration for the running image:
 
 * ``--env``: Any environment variables specified in the ``env`` section will be here. Additional
   ``PAASTA_`` environment variables will also be injected, see the `related docs <yelpsoa_configs.html#marathon-clustername-yaml>`_
-  for more information.
+  for more information. Marathon and Mesos will also set up some
+  `environment variables <https://mesosphere.github.io/marathon/docs/task-environment-vars.html>`_.
```

Comment:
I wouldn't say much more than link to the other thing. The stuff about the conflicting variable is TMI if you ask me.